middleware.js

```diff
@@ -28,65 +33,68 @@ var imports = {};
  * Following that we have a `staticProvider` layer setup to serve the .css
  * files generated by Sass.
  *
- *      var server = connect.createServer(
- *          sass.middleware({
- *              src: __dirname
- *            , dest: __dirname + '/public'
- *          })
- *        , connect.static(__dirname + '/public')
- *      );
+ *   var server = connect.createServer(
+ *     sass.middleware({
+ *       src:  __dirname,
+ *       dest: __dirname + '/public'
+ *     }),
+ *     connect.static(__dirname + '/public')
+ *   );
  *
  * @param {Object} options
  * @return {Function}
  * @api public
  */
 
-module.exports = function(options){
+module.exports = function(options) {
   options = options || {};
 
-  // Accept src/dest dir
-  if ('string' == typeof options) {
+  // Accept single src/dest dir
+  if (typeof options == 'string')
     options = { src: options };
-  }
-
-  // Force compilation
-  var force = options.force || options.response;
 
-  // Debug option
-  var debug = options.debug;
-
-  // Source dir required
-  var src = options.src;
-  if (!src) { throw new Error('sass.middleware() requires "src" directory'); }
-
-  // Default dest dir to source
+  // Source directory (required)
+  var src = options.src || function() {
+    throw new Error('sass.middleware() requires "src" directory.');
+  }();
+  // Destination directory (source by default)
   var dest = options.dest || src;
-
+  // Optional base path for src and dest
   var root = options.root || null;
 
+  // Force compilation everytime
+  var force = options.force || options.response;
+  // Enable debug output
+  var debug = options.debug;
+
   // Default compile callback
-  options.compile = options.compile || function(){
+  options.compile = options.compile || function() {
     return sass;
   };
 
   // Middleware
-  return function sass(req, res, next){
-    if ('GET' != req.method && 'HEAD' != req.method) { return next(); }
+  return function sass(req, res, next) {
+    if (req.method != 'GET' && req.method != 'HEAD')
+      return next();
+
     var path = url.parse(req.url).pathname;
     if (options.prefix && 0 === path.indexOf(options.prefix)) {
       path = path.substring(options.prefix.length);
     }
-    if (/\.css$/.test(path)) {
-      var cssPath = join(dest, path),
+
+    if (!/\.css$/.test(path)) {
+      next();
+    } else {
+      var cssPath  = join(dest, path),
           sassPath = join(src, path.replace(/\.css$/, '.scss')),
-          sassDir = dirname(sassPath);
+          sassDir  = dirname(sassPath);
 
       if (root) {
-        cssPath = join(root, dest, path.replace(new RegExp('^' + dest), ''));
+        cssPath  = join(root, dest, path.replace(new RegExp('^' + dest), ''));
         sassPath = join(root, src, path
             .replace(new RegExp('^' + dest), '')
             .replace(/\.css$/, '.scss'));
-        sassDir = dirname(sassPath);
+        sassDir  = dirname(sassPath);
```

Comment:
I have been reading about it. :astonished:
I'm sorry for this, I had the habit of align all...